alchemy/src/cloudflare/website.ts

```diff
@@ -70,20 +70,41 @@ export async function Website<B extends Bindings>(
   return alchemy.run(id, async () => {
     // building the site requires a wrangler.jsonc file to start
     // - so initialize an empty one if it doesn't exist
-
     const cwd = path.resolve(props.cwd || process.cwd());
-    const fileName =
-      typeof props.wrangler === "boolean"
-        ? "wrangler.jsonc"
-        : typeof props.wrangler === "string"
-          ? props.wrangler
-          : (props.wrangler?.path ?? "wrangler.jsonc");
-    const wranglerPath =
-      fileName && path.relative(cwd, path.join(cwd, fileName));
-    const wranglerMain =
-      typeof props.wrangler === "object"
-        ? (props.wrangler.main ?? props.main)
-        : props.main;
+
+    let fileName;
+    let wranglerPath;
+    let wranglerMain = path.join(cwd, props.main ?? "");
+    // Building fileName and wranglerPath for all the different supporteded cases of props.wrangler
+    // Taking into consideration also cwd
+    if (typeof props.wrangler === "boolean") {
+      fileName = "wrangler.jsonc";
+      wranglerPath = cwd;
+    }
+    if (typeof props.wrangler === "string") {
+      if (path.basename(props.wrangler) === props.wrangler) {
+        fileName = props.wrangler;
+        wranglerPath = cwd;
```

Comment:
What's this case achieving?

Reply:
this check that props.wrangler passed is just the filename with no path: ie 'wrangler.json', in that case that is the fileName, and its pass should be cwd